endpoints/api/billing.py

```diff
@@ -1018,7 +1020,7 @@ def delete(self, orgname, subscription_id):
                 return ("Organization not valid", 400)
 
             model.organization_skus.remove_subscription_from_org(organization.id, subscription_id)
-            return ("Deleted", 204)
+            return ("Deleted", 200)
```

Comment:
I thought so too at first, but there were problems with determining if the SKU was actually removed because the 204 response can't return anything to verify that it was removed. There's an alert I want to show for a successful removal of a SKU